### DISCUSSION TITLE
Why is component's text lighter in color in fluent theme and how to change this?

### DISCUSSION THREAD

**Discussion by alexeyabel** [slint-ui/slint#6593] [General] [answered]
Why is text of components, e.g. the text of a `CheckBox` lighter in color than text of `Text` when using the fluent theme?

![Screenshot_20241020_065819](https://github.com/user-attachments/assets/57888fbe-2bad-448b-8da4-da9f9052aae8)

IMHO, this is a bad design choice because:

1. It can be confused with disabled components.
2. When combining e.g. a `CheckBox` with a `LineEdit` and them some `Text` behind it for a composite option, the colors don't match.
3. It introduces visual noise and the brain needs to process an additional visual category.

I checked and none of the `Palette` colors match the components' text color. Is there a way to change the component text to have the exact same color as the `Text` text?

**Comment by Enyium** [comment, marked as answer]
Already solved, at least for `CheckBox`: #6239. But if this didn't solve it for `Switch` also, you should create an issue for it, referencing the other.

**Comment by alexeyabel** [comment]
I see. It seems the fix was merged, one day after the 1.8.0 release. I will try the dev version.

**Reply by alexeyabel** [reply]
I created an [issue](https://github.com/slint-ui/slint/issues/6595), as you suggested.
